### PR TITLE
Document how to upload larger files via front-end

### LINKED ISSUE #5849: Document how to handle larger file uploads
We could create a brief page in the wiki describing how to configure PHP for larger files, i.e. setting values in the php.in file.

Currently this is documented a few times over in several of the modules that have file uploading features. it would be good to instead centralize this in one place and link to it if necessary.

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Adds instructions to modifying Apache to allow for larger files to be uploaded via the front-end.

These will now be present on Read The Docs.

#### Link(s) to related issue(s)

* Resolves #5849

### CODE REVIEW

**Comment by johnsaigle** [comment]
@driusan @ridz1208 Could you merge this if satisfied?

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Handling Large File Uploads
+
+In order to upload larger files (> 2 MB) for LORUS users, you'll need to configure
```
Comment (driusan) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md]:
```suggestion
In order to upload larger files (> 2 MB) for LORIS users, you'll need to configure
```

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Handling Large File Uploads
+
+In order to upload larger files (> 2 MB) for LORIS users, you'll need to configure
```
Comment (driusan) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md]:
```suggestion
In order for users to upload large files (> 2 MB), you'll need to configure
```

This wording is probably better

File: docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Handling Large File Uploads
+
+In order for users to upload large files (> 2 MB), you'll need to configure
+Apache's `php.ini` configuration file. 
+
+Recommended sample values appropriate for compressed scans not exceeding 500M in size are:
```
Comment (PapillonMcGill) [inline docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md]:
```suggestion
Recommended sample values appropriate for compressed scans not exceeding 500MB in size are: 
```

Comment (driusan) [reply docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md]: @johnsaigle ping.. I tried to click "Commit" on github so I can merge this PR but I don't have permission on the branch.

**Comment by johnsaigle** [comment]
Thanks @PapillonMcGill and @driusan. This should be good to go now.

**Pull request merged**